### ISSUE TITLE
[Suggestion] Angled thrusters

### ISSUE THREAD

**Issue by StarrKiss** [tsunamayo/Starship-EVO#2985]
There are no angled thrusters as of now- which can be quite the frustration when building ships with angled backs (such as stealth fighters. Angled thrusters could serve as thrust in two directions (or just one)- but the main reason I would like them is for aesthetic reasons.

**Comment by Vicomt** [comment]
use a hinge/rotator? Thrusters (block) only contribute towards forward/braking thrust anyway - you'll never go sideways or turn without the smaller brick based thrusters.

**Comment by Gliese-832-c** [comment]
I agree about the hinge/rotator thing, though dedicated angled thruster blocks could save some space on very compact builds.

**Comment by StarrKiss** [comment]
Well- I drew this side diagram- 
![shampooo](https://user-images.githubusercontent.com/13007697/93797455-e546a300-fc01-11ea-90ce-cde9e46d4d2d.png)
and this would mean that the thruster would effectively take FOUR blocks- eliminating any thin angle thrusters- Plus- this is hard to do for curves BESIDES 45 degrees to match it perfectly withs ide walls.

**Comment by Gliese-832-c** [comment]
Now that I think about it, I agree. We should get a thruster wedge block that can be stretched and has a seamless texture to neighboring thruster wedge blocks.

**Comment by ProPeach** [comment]
Seamless textures between existing thruster blocks would be brilliant either way, the seam can be very ugly if you need to use more than two thruster blocks (which is a common event, any non-quadrilateral shape requires at least two stretched thrusters).

**Comment by tsunamayo** [comment]
Yeah a angled thruster would be cool.
I think Zach or Xerma made a suggestion about this a while ago.
I dont see myself doing a seamless thruster though, the tech is not suited for this. 
Thanks

**Comment by Garrett-C** [comment]
> I dont see myself doing a seamless thruster though, the tech is not suited for this.

I think it's fair enough that the tech doesn't really allow for the thrusters to be seamless perhaps we could have a glow block instead that we could put on top of the engines that could create the look of a merged engine. I mean realistically the engine merging is only an issue on large engines or non square engines and these will likely be in some form of block or brick housing anyway so a flat coloured glow block would probably work quite well to give the same effect.

**Comment by ProPeach** [comment]
That's a shame about the seamless thrusters, I thought it could work similar to the bevels on hull blocks. I don't think there are any solutions that are as elegant as that

**Comment by Garrett-C** [comment]
> That's a shame about the seamless thrusters, I thought it could work similar to the bevels on hull blocks. I don't think there are any solutions that are as elegant as that

Well, the other option is providing a way to get the thruster look with an easily tileable block. A glow block or brick could achieve this nicely.

**Comment by Hecateus** [comment]
@Garrett-C Glow Blocks as 'Thrusters' is a big thing on the fancier builds in Avorion.
@StarrKiss I think you mean Gimballed thrusters, given we have gimballed guns this may be doable even if the only obvious change was with where the engine wash was pointing.  A single block/brick of multidirectional locked RCS thrusters would simplify builds too.

**Comment by StarrKiss** [comment]
> @Garrett-C Glow Blocks as 'Thrusters' is a big thing on the fancier builds in Avorion.
> @StarrKiss I think you mean Gimballed thrusters, given we have gimballed guns this may be doable even if the only obvious change was with where the engine wash was pointing.  A single block/brick of multidirectional locked RCS thrusters would simplify builds too.

I do not mean gimbled thrusters, I mean normal fixed thsuters at an angle.

**Comment by WILDCATreactor** [comment]
> > I dont see myself doing a seamless thruster though, the tech is not suited for this.
> 
> I think it's fair enough that the tech doesn't really allow for the thrusters to be seamless perhaps we could have a glow block instead that we could put on top of the engines that could create the look of a merged engine. I mean realistically the engine merging is only an issue on large engines or non square engines and these will likely be in some form of block or brick housing anyway so a flat coloured glow block would probably work quite well to give the same effect.

I can see merit for getting Large Round Thrusters, at the very least.
